server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/environment/EnvironmentService.java

```diff
@@ -155,6 +156,18 @@ public Environment detail(@NonNull Long id) {
         return innerDetail(id);
     }
 
+    @Transactional(rollbackFor = Exception.class)
+    @SkipAuthorize("odc internal usage")
+    public List<Environment> getByIdIn(@NotNull Collection<Long> ids) {
+        List<EnvironmentEntity> entity = environmentRepository.findByIdIn(ids);
```

Comment:
if ids is an empty collection, the JPA query will failed.

may add empty check, or just return empty list while ids is empty.

Reply:
got it

Reply:
This part of the logic has been integrated into the list.